### DISCUSSION TITLE
Allegro and ASE

### DISCUSSION THREAD

**Discussion by schiotz** [mir-group/allegro#18] [Q&A] [answered]
Hi folks,

NequIP has both a LAMMPS and an ASE interface for e.g. molecular dynamics.  The Allegro page only mentions LAMMPS, but is it also possible to run the model using ASE?  Is it perhaps just the same driver as NequIP since Allegro is built on top of NequIP?

Best regards

Jakob

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi Jakob,

> Is it perhaps just the same driver as NequIP since Allegro is built on top of NequIP?

Yep, an Allegro model is for all purposes another `nequip` model (the framework, not the architecture) and can be used with all the same tools and adapters.  The only distinction comes in with the LAMMPS plugin due to the additional parallel abilities of `pair_allegro` that `pair_nequip` lacks.

(As a result Allegro models can _also_ be run in LAMMPS using `pair_nequip`, as long as you use only one rank and one GPU.)
